canonical/ubuntu.com
# fix(redirect): restored some broken redirects

## Done

- Fixed broken redirects:
  - [developer/core/get-started/installation-medias](https://ubuntu-com-13833.demos.haus/developer/core/get-started/installation-medias) -> /download/iot/installation-media
  - [developer/core/get-started/raspberry-pi-2-3](https://ubuntu-com-13833.demos.haus/developer/core/get-started/raspberry-pi-2-3) -> /download/raspberry-pi
  - [download/iot/intel-iotg](https://ubuntu-com-13833.demos.haus/download/iot/intel-iotg) -> /download/iot/intel-iot
  - [engage/iot-elkhart-lake](https://ubuntu-com-13833.demos.haus/engage/iot-elkhart-lake) -> /download/iot/intel-iot

## QA

- Check out this feature branch
- Run the site using the command `./run serve` or `dotrun`
- View the site locally in your web browser at: http://0.0.0.0:8001/
    - Be sure to test on mobile, tablet and desktop screen sizes
- Check that the redirects listed in the "Done" section above do not point to a 404 page

## Help

[QA steps](https://discourse.canonical.com/t/qa-steps/152) - [Commit guidelines](https://discourse.canonical.com/t/commit-guidelines/148)
